webpack/assets/javascripts/react_app/components/CPUCoresInput/cpuCoresInput.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../common/colors';
+
+.form-group {
```

Comment:
It will affect other components as it is too generic.
Also, do we actually need it now when you use the react grid?